test/ClangImporter/pcm-emit-system-module.swift

```diff
@@ -0,0 +1,8 @@
+// Emit the explicit system module.
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-emit-pcm -module-name CoreCooling -Xcc -Xclang -Xcc -emit-module -Xcc -Xclang -Xcc -fsystem-module -o %t/CoreCooling.pcm %S/Inputs/custom-modules/module.map
+
+// Verify that the input modulemap if marked [System] in the output of the -dump-pcm action.
+// RUN: %swift-dump-pcm %t/CoreCooling.pcm | %FileCheck %s --check-prefix=CHECK-SYSTEM-INPUT
+// CHECK-SYSTEM-INPUT: Input file: SOURCE_DIR/{{.*}}/ClangImporter/Inputs/custom-modules/module.map [System]
```

Comment:
Nice—I didn't realize the dump output distinguished them that way. That's more direct than my suggestion!